src/sentry/migrations/0268_rename_issue_widget_query_fields.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 2.2.24 on 2021-09-23 14:02
+from django.db import migrations
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def rename_issue_widget_query_fields(apps, schema_editor):
+
+    DashboardWidgetQuery = apps.get_model("sentry", "DashboardWidgetQuery")
+
+    old_to_new_field_mapping = {
+        "count": "events",
+        "userCount": "users",
+        "lifetimeCount": "lifetimeEvents",
+        "lifetimeUserCount": "lifetimeUsers",
+    }
+
+    for query in RangeQuerySetWrapperWithProgressBar(DashboardWidgetQuery.objects.all()):
+        fields = getattr(query, "fields")
```

Comment:
This `getattr` seems unnecessary, can we just access `query.fields` directly?

Reply:
updated! also spoke to Will and we're ok with keeping this migration as not dangerous